libs/environment/src/utils/validate-node.tsx

```diff
@@ -40,24 +40,15 @@ export const getIsNodeDisabled = (env: Networks, data?: NodeData) => {
 
 export const getIsFormDisabled = (
   currentNode: string | undefined,
-  inputText: string,
   env: Networks,
   state: Record<string, NodeData>
 ) => {
   if (!currentNode) {
     return true;
   }
 
-  if (
```

Comment:
question: should we add/remove/alter tests for this behaviour change? Or is this currently not united tested?

Reply:
the react component itself is not tested, that's where this is being called. most cases regarding functionality are covered in the hook tests, but it might make sense to do some ui tests for the react component, too.

Reply:
added a bunch of tests for the node switcher component @dexturr